views/beers.hbs

```diff
@@ -0,0 +1,10 @@
+<h1>Beers</h1>
+<div class="beer-container">
+
+{{#each beersFromApi}}
+<img src="{{this.image_url}}" alt="beer">
```

Comment:
Some beers do not have images, there could be some logic here that checks for this